src/Sql/Sql/help/Az.Sql.md

```diff
@@ -117,7 +117,8 @@ Gets the geo-replication links between an Azure SQL Database and a resource grou
 Retrieves the distinct restore points from which a SQL Data Warehouse can be restored.
 
 ### [Get-AzSqlDatabaseSecureConnectionPolicy](Get-AzSqlDatabaseSecureConnectionPolicy.md)
-Gets the secure connection policy for a database.
+Gets the secure connection policy for a database. 
```

Comment:
do we have cmdlet for server connection policy too?

Reply:
no